pkg/config/model.go

```diff
@@ -91,6 +91,9 @@ type Meta struct {
 
 	// A sequence number representing a specific generation of the desired state. Populated by the system. Read-only.
 	Generation int64 `json:"generation,omitempty"`
+
+	// FullName is the name in resource including namespace and expanded GVK. Typically used for logging and metadata.
+	FullName string
```

Comment:
I am concerned about this.
this is not an metadata, and we are building it for all kinds of objects.

Reply:
Does not object need this? Or only a subset like VS and DR?

Reply:
+1 to concerns. I like the idea since I know generating this is expensive. However, i am a bit worried about precomputing things like this. Way to easy to accidentally forget to set it. wonder if some sort of on-demand way we can?

Reply:
And yes its only DR and VS

Reply:
> Way to easy to accidentally forget to set it. wonder if some sort of on-demand way we can?

That is another thought I had. Add FullName to ConfigMeta struct and generate once on first use and reuse later. Would that work?

Reply:
Made few changes. See if this version is better

Reply:
I will fix UTs if this is a better approach